src/pilot/constants.patch

```diff
@@ -1,11 +1,19 @@
---- constants.py	2018-09-19 04:11:26.000000000 -0500
-+++ constants_backup.py	2019-01-08 12:18:12.983836000 -0600
-@@ -20,6 +20,8 @@
- DEFAULT_WSMAN_SSL_ERROR_RETRIES = 3
- DEFAULT_WSMAN_SSL_ERROR_RETRY_DELAY_SEC = 0
+--- constants.py.orig	2019-01-23 14:21:10.077000000 +0000
++++ constants.py	2019-01-23 13:45:52.275000000 +0000
+@@ -75,3 +75,16 @@
+                 cls.reboot_forced_shutdown]
  
-+NOT_SUPPORTED_MSG = " operation is not supported on th"
```

Comment:
The original constants.patch patched in the NOT_SUPPORTED_MSG constant, which the dracclient/resources/raid.py is dependent on.  The new patch does not patch that in.  We need to continue to patch in the NOT_SUPPORTED_MSG constant, or things will break.

Reply:
Actually, I can see that the NOT_SUPPORTED_MSG constant is available in OSP13z4, so monkey patching it in is no longer needed.  Please ignore my above comment.